### PR TITLE
[zh] Sync ist0163 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync ist0163 into Chinese:
```
content/zh/docs/reference/config/analysis/ist0163/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by hanxiaop — APPROVED** [review]

File: content/zh/docs/reference/config/analysis/ist0163/index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+您将收到此消息：
+
+{{< text plain >}}
+Warning [IST0163] (MutatingWebhookConfiguration istio-sidecar-injector-external-istiod testing.yml:28) The hostname () that was provided for the webhook (rev.namespace.sidecar-injector.istio.io) to reach the ingress gateway on the external control plane cluster is blank. Traffic may not flow properly.
+Warning [IST0163] (ValidatingWebhookConfiguration istio-validator-external-istiod testing.yml:1) The hostname () that was provided for the webhook (rev.validation.istio.io) to reach the ingress gateway on the external control plane cluster is blank. Traffic may not flow properly.
+{{< /text >}}
+
+当您的集群具有以下缺少 Webhook URL 的 `ValidatingWebhookConfiguration`
+和 `MutatingWebhookConfiguration`（为清楚起见而缩短）时：
```
Comment (windsonsea) [inline content/zh/docs/reference/config/analysis/ist0163/index.md:22]:
```suggestion
当集群的 `ValidatingWebhookConfiguration` 和 `MutatingWebhookConfiguration`
字段（为清楚起见而缩短）缺少 Webhook URL 时：
```

File: content/zh/docs/reference/config/analysis/ist0163/index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+您将收到此消息：
+
+{{< text plain >}}
+Warning [IST0163] (ValidatingWebhookConfiguration istio-validator-external-istiod testing.yml:1) The hostname (https://thisisnotarealdomainname.com:15017/validate) that was provided for the webhook (rev.validation.istio.io) to reach the ingress gateway on the external control plane cluster cannot be resolved via a DNS lookup. Traffic may not flow properly.
+{{< /text >}}
+
+当您的集群具有以下 `ValidatingWebhookConfiguration`
+和 `MutatingWebhookConfiguration`（为清楚起见而缩短）时，
+它们使用的主机名在 DNS 查找期间无法被解析：
```
Comment (windsonsea) [inline content/zh/docs/reference/config/analysis/ist0163/index.md:95]:
```suggestion
当集群的 `ValidatingWebhookConfiguration` 和 `MutatingWebhookConfiguration`
（为清楚起见而缩短）字段所使用的主机名在 DNS 查询期间无法被解析时：
```
google 中 `DNS 查询`的词频更高

File: content/zh/docs/reference/config/analysis/ist0163/index.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+有多种方法可以解决这些无效配置，具体取决于配置无效的原因。
+
+如果您的 Webhook 配置未定义 URL，
+则添加使用主机名的有效 URL 将解决此警告消息。
+有关如何执行此操作的说明可以在[此处](/zh/docs/setup/install/external-controlplane/#set-up-the-remote-config-cluster)找到。
+
+如果您的主机名无法通过 DNS 查找解析为 IP 地址，
+您可以尝试在本地计算机上运行 `dig <your-hostname>`
+来查看是否触发 DNS 解析。如果您的本地计算机可以通过 DNS 查找解析主机名，
+您的集群可能不行。任何阻止 DNS 流量的安全规则都可能导致解析查找失败。
```
Comment (windsonsea) [inline content/zh/docs/reference/config/analysis/ist0163/index.md:171]:
```suggestion
那可能是您的集群不能解析此主机名。任何阻止 DNS 流量的安全规则都可能导致解析查询失败。
```
解析查询 的词频更高

File: content/zh/docs/reference/config/analysis/ist0163/index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+有多种方法可以解决这些无效配置，具体取决于配置无效的原因。
+
+如果您的 Webhook 配置未定义 URL，
+则添加使用主机名的有效 URL 将解决此警告消息。
+有关如何执行此操作的说明可以在[此处](/zh/docs/setup/install/external-controlplane/#set-up-the-remote-config-cluster)找到。
+
+如果您的主机名无法通过 DNS 查找解析为 IP 地址，
+您可以尝试在本地计算机上运行 `dig <your-hostname>`
+来查看是否触发 DNS 解析。如果您的本地计算机可以通过 DNS 查找解析主机名，
```
Comment (windsonsea) [inline content/zh/docs/reference/config/analysis/ist0163/index.md:170]:
```suggestion
来查看是否触发 DNS 解析。如果您的本地计算机可以通过 DNS 查询来解析主机名，
```

File: content/zh/docs/reference/config/analysis/ist0163/index.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
+---
+title: InvalidExternalControlPlaneConfig
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当为外部控制平面上的入口网关提供的地址无效时，会出现此消息。
+该地址可能因多种原因而无效，包括：主机名地址格式错误、
+主机名无法通过 DNS 查找解析为 IP 地址，或者主机名解析为零个 IP 地址。
```
Comment (windsonsea) [inline content/zh/docs/reference/config/analysis/ist0163/index.md:10]:
```suggestion
主机名无法通过 DNS 查询解析为 IP 地址，或者主机名解析为零个 IP 地址。
```
google 中 `DNS 查询`的词频貌似更高

**Review by windsonsea — COMMENTED** [review]
Some text suggestions

**Review by windsonsea — APPROVED** [review]
Thanks, lgtm.

**Pull request merged**